x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/distributed_lock_manager/distributed_lock_manager.spec.ts

```diff
@@ -486,24 +486,41 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
       const LOCK_ID = 'my_lock_with_lock';
 
       describe('Successful execution and concurrent calls', () => {
-        let executions: number;
```

Comment:
Not related but this test was flaky (failed 1 out of 100 runs). The changes here should improve it - still waiting for flaky test runner to complete.